x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/trigger/schedule/CronScheduleTests.java

```diff
@@ -54,18 +58,25 @@ public void testParseMultiple() throws Exception {
         assertThat(crons, hasItemInArray("0 0/3 * * * ?"));
     }
 
+    public void testMultipleCronsNextScheduledAfter() {
```

Comment:
should we add an expired cron entry to the list here too just to make sure?

Reply:
done